sdk/examples/wallet/spammer.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! In this example we will spam transactions from multiple threads simultaneously to our own address.
+//!
+//! `cargo run --example threads --release`
+
+use std::{env::var, time::Duration};
+
+use iota_sdk::{
+    client::{
+        constants::SHIMMER_COIN_TYPE,
+        request_funds_from_faucet,
+        secret::{mnemonic::MnemonicSecretManager, SecretManager},
+    },
+    wallet::{account::types::AccountAddress, Account, ClientOptions, Result, SendAmountParams, Wallet},
+};
+use tokio::{task::JoinSet, time::sleep};
+
+// The account alias used in this example.
+const ACCOUNT_ALIAS: &str = "spammer";
+// The number of spamming rounds.
+const NUM_ROUNDS: usize = 1000;
+// The amount to send in each transaction.
+const SEND_AMOUNT: u64 = 1_000_000;
+// The number of concurrent transactions.
+const NUM_CONCURRENT_TXS: usize = 32;
+
+#[tokio::main(flavor = "multi_thread")]
+async fn main() -> Result<()> {
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
+
+    println!(
+        "{} cores detected. Running spammer with {NUM_CONCURRENT_TXS} concurrent transactions...",
+        num_cpus::get(),
+    );
```

Comment:
What's the point of printing the amount of cores when we just use the NUM_CONCURRENT_TXS anyways?

Reply:
hmm, I see your point. My intention was to let the user decide how many cores to use and at least give the information what  the reasonable max number COULD be used. You never know  what other stuff is currently running on the device this example is running on, so the example shouldn't smoke the CPU. I have a better idea of doing this now. Good catch!

Reply:
Changed.